packages/traefik-dynamic/consul-template-traefik

```diff
@@ -0,0 +1,2 @@
+consul	ALL=(ALL)	NOPASSWD:	/usr/bin/systemctl restart traefik.service
+
```

Comment:
That will be taken care of by yum/the package configuration, which runs 
as root and so doesn’t need special permission. Check out the upgrade 
steps in `spec.yml`